setup/install.sh

```diff
@@ -122,6 +122,15 @@ then
   exit
 fi
 
+# check for .ssh folder
+doil_check_ssh
+if [[ $? -ne 0 ]]
+then
+  doil_status_failed
+  doil_status_send_error "REQUIREMENT ERROR" "Missing .ssh folder!\n\tThis folder is required by doil.\n\tIf you only use public github repositories please create\n\tan empty .ssh folder in your home directory to fit this doil condition.\n\tOtherwise you should create ssh keys and link them with your private git repositories.\n\tFollow this link https://docs.github.com/de/authentication/connecting-to-github-with-ssh/about-ssh"
```

Comment:
Maybe add a comma after "public github repositories" for better readability.
The last sentence could be "Follow this link ... for more information." to make it a bit clearer.

Reply:
Thats right, thank you for this hints.

Reply:
Done!